plugins/rest/komand_rest/util/util.py

```diff
@@ -61,6 +62,34 @@ def first(sequence, default=""):
     return next((x for x in sequence if x), default)
 
 
+def convert_dict_body_to_string(dict_object: Dict[str, Any]) -> str:
+    """
+    This method will convert a dict object to a string
+    suitable for sending data in x-www-form-urlencoded format
+    :param dict_object: The dict object to convert
+    :return: A new string properly formatted
+    """
+    output_string = ""
+    for key, value in dict_object.items():
+        output_string += f"{key}={value}&"
+    return output_string[:-1]
```

Comment:
You can substitute this function with `urllib.parse.urlencode()`: https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlencode